src/main/java/net/fabricmc/accesswidener/AccessWidenerVisitor.java

```diff
@@ -20,6 +20,14 @@
  * A visitor of the entries defined in an access widener file.
  */
 public interface AccessWidenerVisitor {
+	/**
+	 * Visits the version.
+	 *
+	 * @param version the access widener's version
+	 */
+	default void visitVersion(int version) {
+	}
```

Comment:
No, the version shouldnt be passed around here, the version only relates to the format and thus the reader/writer. This has already discussed in great detail on some of the recent PRs that made up 2.0